lib/ui/sign_transaction/interactor/data/transaction_action_data.dart

```diff
@@ -58,4 +58,12 @@ class TransactionDetailsCardData {
   });
 
   String get userFriendlyContractAction => knownContractActions[contractAction] ?? contractAction;
+  String? get requestor => params['requestor'];
```

Comment:
not sure ill need this. Next pr will work on top of this

Reply:
Not sure what's a requester?